ukm-semantics/main/preprocessing/methods.md

```diff
@@ -26,6 +26,21 @@ module UKM-PREPROCESSING-METHODS
         <method-name> Method </method-name>
         <method-outer-attributes> Atts:OuterAttributes </method-outer-attributes>
         requires getEndpointName(Atts, MethodIdentifier) =/=K ""
+            andBool getStorageName(Atts) ==K ""
+        [priority(50)]
+    rule
+        <k>
+            ukmPreprocessMethod(_Trait:TypePath, MethodIdentifier:Identifier, Method:PathInExpression)
+            => ukmPreprocessStorage
+                (... fullMethodPath: Method
+                , storageName: getStorageName(Atts)
+                )
+            ...
+        </k>
+        <method-name> Method </method-name>
+        <method-outer-attributes> Atts:OuterAttributes </method-outer-attributes>
+        requires getStorageName(Atts) =/=K ""
+            andBool getEndpointName(Atts, MethodIdentifier) ==K ""
         [priority(50)]
```

Comment:
Adding a priority of 50 isn't necessary, if I'm not mistaken. That is the default priority of all rules.

Reply:
Removed.

I can never decide if it's a good idea to add the priority attribute to tell people that they must pay attention because this rule interacts with rules of different priorities... I guess that this matters most when there are higher priority rules that may apply, since it would help people figure out why the current rule does not apply. However, this is not what we have here.